locomotion-sdk/src/context/onboarding/index.js

```diff
@@ -70,7 +70,7 @@ const authContainer = () => {
 
       if (vertResponse.status !== 'OK' || !vertResponse.refreshToken || !vertResponse.accessToken) {
         console.log('Bad vert with response', vertResponse);
-        return false;
+        return 'failed';
```

Comment:
Why you changed it?

Reply:
need to check that it returned false to show error but even if theres no error the function returns nothing so it came back false and showed error, so i switched it to check if returns failed

Reply:
So add return true in the end of the function